platform/viewer/public/config/default.js

```diff
@@ -30,10 +30,15 @@ window.config = {
         // wadoUriRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/wado',
         // qidoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
         // wadoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
+
         // new server
-        wadoUriRoot: 'https://domvja9iplmyu.cloudfront.net/dicomweb',
-        qidoRoot: 'https://domvja9iplmyu.cloudfront.net/dicomweb',
-        wadoRoot: 'https://domvja9iplmyu.cloudfront.net/dicomweb',
+        //wadoUriRoot: 'https://domvja9iplmyu.cloudfront.net/dicomweb',
+        //qidoRoot: 'https://domvja9iplmyu.cloudfront.net/dicomweb',
+        //wadoRoot: 'https://domvja9iplmyu.cloudfront.net/dicomweb',
+        wadoUriRoot: 'http://localhost/dicom-web',
+        qidoRoot: 'http://localhost/dicom-web',
+        wadoRoot: 'http://localhost/dicom-web',
```

Comment:
wadoUriRoot is redundant at this point unless it is actually using wadoUri

Reply:
This was a temporary config for local dicomweb server. Changed in the next commit